datajoint/expression.py

```diff
@@ -661,10 +661,10 @@ def cursor(self, offset=0, limit=None, order_by=None, as_dict=False):
         """
```

Comment:
`cursor` is only used internally in datajoint, so it's best to handle the sorting logic in the calling context as appropritate to each case, taking into consideration the existing `top` restruction.